go.mod

```diff
@@ -296,3 +296,5 @@ replace (
 	github.com/insomniacslk/dhcp => github.com/elastic/dhcp v0.0.0-20200227161230-57ec251c7eb3 // indirect
 	github.com/tonistiigi/fifo => github.com/containerd/fifo v0.0.0-20190816180239-bda0ff6ed73c
 )
+
+replace github.com/apoydence/eachers => github.com/poy/eachers v0.0.0-20181020210610-23942921fe77
```

Comment:
Does go.mod support comments? If yes, likely worth to add on here.

Reply:
@ruflin I am not sure that comment would stay, it's a mostly generated file, I think everything in that files is subject to be regenerated by a tool or `go mod tidy`

Concerning contributing changes upstream, It's something we could do but before I think we should look at our usage of cloudfoundry's noaa, last changes into that library is 5 years ago and we included it 2 years ago. Which is an eternity in code life.

@jsoriano I believe you have implemented the cloudfoundry support a few years ago and you make use of noaa's client, I wonder if this is still the valid way to do in 2022?

Reply:
> Concerning contributing changes upstream, It's something we could do but before I think we should look at our usage of cloudfoundry's noaa, last changes into that library is 5 years ago and we included it 2 years ago. Which is an eternity in code life.
>
> @jsoriano I believe you have implemented the cloudfoundry support a few years ago and you make use of noaa's client, I wonder if this is still the valid way to do in 2022?

Yeah, this noaa library is used to consume from the v1 API version of the Cloudfoundry firehose, this version has been deprecated for years, but is (or used to be) more reliable in most deployments than the current v2 API, this is why it is still used, and why we implemented support for this version, and made it the default, after having support for v2.

We'd have to check if the situation has changed during this time, but in the meantime I think that we can continue using noaa as is.